src/System.Private.CoreLib/shared/System/Reflection/LocalVariableInfo.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Reflection
+{
+    public class LocalVariableInfo
+    {
+#if CORECLR
```

Comment:
Hmm, this ifdef does not look nice.

It may look better to make the public LocalVariableInfo look like the CoreRT version with no fields. And introduce a new internal RuntimeLocalVariableInfo type with these fields that will inherit from the public LocalVariableInfo and that the VM will create and fill in.